noxfile.py

```diff
@@ -26,7 +26,7 @@
 
 DEFAULT_PYTHON_VERSION = "3.8"
 SYSTEM_TEST_PYTHON_VERSIONS = ["3.8"]
-UNIT_TEST_PYTHON_VERSIONS = ["3.6", "3.7", "3.8", "3.9"]
+UNIT_TEST_PYTHON_VERSIONS = ["3.8", "3.9", "3.10"]
```

Comment:
Should we call out in the release notes that Python 3.6 and 3.7 are no longer supported?

Reply:
Sure thanks, let me add